docs/addon-dashboard.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: "Dashboard addon"
+permalink: /docs/addon-dashboard
+---
+
+# Dashboard addon
+
+The standard Kubernetes Dashboard is a convenient way to keep track of the
+activity and resource use of MicroK8s
+
+To log in to the Dashboard, you will need the access token (unless RBAC has
+also been enabled). This is generated randomly on deployment, so a few commands
+are needed to retrieve it:
```

Comment:
We have to retreive the token after the addon gets enabled. Contrary to the docs for other addons (eg fluentd) here we do not say that the dashboard is enabled with `microk8s.enable dashboard`.

Reply:
ah yes, good point I will fix that

Reply:
done